core/src/main/scala/cats/data/XorT.scala

```diff
@@ -22,6 +22,13 @@ final case class XorT[F[_], A, B](value: F[A Xor B]) {
 
   def getOrElse[BB >: B](default: => BB)(implicit F: Functor[F]): F[BB] = F.map(value)(_.getOrElse(default))
 
+  def getOrElseF[BB >: B](default: => F[BB])(implicit F: Monad[F]): F[BB] = {
+    F.flatMap(value) {
+      case Xor.Left(a) => default
+      case Xor.Right(a) => F.pure(a)
```

Comment:
Extremely minor: I'd be inclined to use the following names for the captured values:

```scala
case Xor.Left(_) => default
case Xor.Right(b) => F.pure(b)
```

As this doesn't actually result in any difference, I'm not sure I can really justify the nitpick though...